src/features/rewards/tab/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledSlider,
+  StyledBullet,
+  StyledTextOne,
+  StyledTextTwo,
+  StyledSwitch
+} from './style'
+
+export interface Props {
+  id?: string
```

Comment:
Can we use `testId` and `data-test-id` attribute or did I miss a change in process for that?

Reply:
done